src/test/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActorTest.scala

```diff
@@ -143,10 +145,8 @@ class TaskLauncherActorTest extends AkkaUnitTest with Eventually {
       Mockito.when(instanceTracker.instancesBySpecSync).thenReturn(InstanceTracker.InstancesBySpec.forInstances(newInstances))
       launcherRef ! InstanceUpdated(newInstance, None, Seq.empty)
 
-      Then("the actor requeries the backoff delay")
+      Then("the actor re-queries the backoff delay")
       rateLimiterActor.expectMsg(RateLimiterActor.GetDelay(upgradedApp.configRef))
-      val newDelay: Timestamp = clock.now() + 5.seconds
-      rateLimiterActor.reply(RateLimiter.DelayUpdate(upgradedApp.configRef, Some(newDelay)))
```

Comment:
This is the last message in the test and looks like we are NOT (correct me if I missed something) checking if `launcherRef` actually received the `DelayUpdate` message from `RateLimiterActor`. Hence, I deleted this line.